src/assets/scripts/client/InputController.js

```diff
@@ -307,6 +307,9 @@ export default class InputController {
         this.input.callsign = aircraftModel.callsign;
         this.input.command = '';
         this.$commandInput.val(`${aircraftModel.callsign} `);
+        if (!this.$commandInput.is(':focus')) {
```

Comment:
There should be an empty line above this `if`, and also an empty line after the closing `}`

Reply:
Indeed it should.